src/components/@layout/Container/Container.stories.tsx

```diff
@@ -4,14 +4,87 @@ import { ComponentStory, ComponentMeta } from '@storybook/react';
 import Container from '.';
 
 export default {
-  title: 'Container',
+  title: 'Design System/Layout/Container',
   component: Container,
   parameters: {
     layout: 'fullscreen',
+    componentSubtitle:
+      'Container는 박스 사이즈 안에서 children이 표시될 수 있도록 감쌉니다.',
+  },
+  argTypes: {
+    children: {
+      name: 'children',
+      description: 'Container 내부에 위치하는 요소입니다.',
+      table: {
+        type: { summary: 'ReactNode' },
+      },
+    },
+    overflowX: {
+      name: 'overflowX',
+      description:
+        '너비를 초과하는 요소가 존재할 경우 어떻게 처리할 것인지 결정합니다.',
+      table: {
+        type: { summary: 'string' },
+      },
+      type: { name: 'string', required: true },
+      defaultValue: 'hidden',
+      control: {
+        type: 'select',
+        options: [
+          'hidden',
+          'visible',
+          'clip',
+          'auto',
+          'inherit',
+          'initial',
+          'unset',
+        ],
+      },
+    },
+    overflowY: {
+      name: 'overflowY',
+      description:
+        '높이를 초과하는 요소가 존재할 경우 어떻게 처리할 것인지 결정합니다.',
+      table: {
+        type: { summary: 'string' },
+      },
+      type: { name: 'string', required: true },
+      defaultValue: 'hidden',
+      control: {
+        type: 'select',
+        options: [
+          'hidden',
+          'visible',
+          'clip',
+          'auto',
+          'inherit',
+          'initial',
+          'unset',
+        ],
+      },
+    },
   },
```

Comment:
```suggestion
    overflowX: {
      name: 'overflowX',
      description:
        '너비를 초과하는 요소가 존재할 경우 어떻게 보여줄 것인지 결정합니다.',
      table: {
        type: { summary: "CSSProperties['overflowX']" },
      },
      type: { name: 'string', required: true },
      defaultValue: 'hidden',
      control: {
        type: 'select',
        options: ['hidden', 'visible', 'clip', 'scroll', 'auto'],
      },
    },
    overflowY: {
      name: 'overflowY',
      description:
        '높이를 초과하는 요소가 존재할 경우 어떻게 보여줄 것인지 결정합니다.',
      table: {
        type: { summary: "CSSProperties['overflowY']" },
      },
      type: { name: 'string', required: true },
      defaultValue: 'hidden',
      control: {
        type: 'select',
        options: ['hidden', 'visible', 'clip', 'scroll', 'auto'],
      },
    },
  },
```

타입 추론이 안되는 것들은 이렇게 명시적으로 넣어주는 것도 괜찮아보여요.

그리고 inherit은 상대적인 값이고 initial은 (명세에서는 visible이지만) 이미 default hidden으로 제공하고 있어서, select로 제공하는 속성들은 키워드 정도만 와도 될 것 같다고 생각해요. MDN 키워드 밸류 기준으로 가져왔어요!

https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-xhttps://developer.mozilla.org/en-US/docs/Web/CSS/overflow-x

Reply:
수정했습니다.
감사해요!